### ISSUE TITLE
Some NCBIGene records should have biolink category "GenomicEntity"

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#1220]
Example: `NCBIGene:780896`.

<img width="800" alt="Screen Shot 2021-02-02 at 4 44 00 PM" src="https://user-images.githubusercontent.com/5562107/106682298-08ea0480-6577-11eb-8ed2-af37e6bc4a3a.png">

Note that under the "General Gene Information" section, it has the property "phenotype only". I think in that case it makes sense to give it a biolink category `biolink:GenomicEntity` which is our closest Biolink approximation for "locus". I think this would require a fix in `ncbigene_to_tsv_to_kg_json.py`.

See issue #1213 which first brought this issue to my attention.

**Comment by saramsey** [comment]
In addition to correctly setting the biolink category, we should prepend "Genetic locus for " to the `name` and `full_name` fields for these types of NCBIGene records.

**Comment by kvarforl** [comment]
looks like this change (for the most part) made it into kg2.5.2.
```
match (n{id:"NCBIGene:780896"}) return n.name, n.full_name, n.category_label
```

n.name | n.full_name | n.category_label
-- | -- | --
"ACTD" | "Genetic locus associated with Acetabular dysplasia" | "genomic_entity"

@saramsey the prepending of "Genetic locus for" made it into `full_name` but not `name`. good enough, or additional tweaks required?

**Comment by kvarforl** [comment]
@dkoslicki @edeutsch  can you take a look at the above comment and let me know if you would like additional adjustments?

**Comment by edeutsch** [comment]
Thanks! I think we wanted the prefix "Genetic locus associated with " on the `name` field as well. The issue is that the NodeSynonymizer uses the `name` field to associate things, and was conflating the disease with the locus because they have the same name. The NS could still compensate for this, but fixing it at the KG level seemed to make the most sense?

**Comment by kvarforl** [comment]
Okay, I've added a fix for this! It's in https://github.com/RTXteam/RTX/commit/aacf9c265f1170136cebeedf4c73555764c45106 ; I mistyped the issue number in my commit message, hence the lack of traditional linkage :)

this should appear in the next kg2 build.

**Comment by saramsey** [comment]
Great, looks like there is nothing for me to do here, as @edeutsch has communicated what specifically is needed (i.e., on the `name` field and not just on the `full_name` field).

**Comment by ericawood** [comment]
Also, I think some NCBIGene records should have the biolink category "microRNA":

Ex.
```
{
  "iri": "https://identifiers.org/ncbigene:100126330",
  "synonym": [
    "MIR941-4",
    "MIRN941-4",
    "hsa-mir-941-4"
  ],
  "category_label": "gene",
  "full_name": "microRNA 941-4",
  "deprecated": "False",
  "name": "MIR941-4",
  "description": "Type:ncRNA; Locus:20q13.33; NameStatus:official",
  "provided_by": "identifiers_org_registry:ncbigene",
  "id": "NCBIGene:100126330",
  "category": "biolink:Gene",
  "update_date": "20210302"
}
```

This is labeled as a gene, but it is a microRNA (it's xref is `mirbase:MI0005766` from the ETL of miRBase #1247). Is this a problem?

**Comment by ericawood** [comment]
The output of the cypher:
`match (n {provided_by: 'identifiers_org_registry:ncbigene'}) where lower(n.full_name) starts with "microrna" and lower(n.category_label)="gene" return count(n)`

is 1915.

**Comment by edeutsch** [comment]
I think it hinders our reasoning on KG2, yes, so a fix would be appreciated!

**Comment by ericawood** [comment]
The microRNA issue is addressed in 3378f83e9e057a437bbd3e5223ab1b3b0f1e093f (forgot to tie that commit to this issue)

**Comment by kvarforl** [comment]
Okay looks like the naming portion of this is fixed in kg2.6.0:

n.name | n.full_name | n.category_label
-- | -- | --
"Genetic locus associated with ACTD" | "Genetic locus associated with Acetabular dysplasia" | "genomic_entity"

closing it out, as I think 1379 covers the microRNA stuff :)
